snuba/clusters/cluster.py

```diff
@@ -65,6 +78,17 @@ def get_clickhouse_ro(self) -> ClickhousePool:
     def get_reader(self) -> Reader[SqlQuery]:
         return self.__reader
 
+    def get_writer(
+        self,
+        table_name: str,
```

Comment:
Here is the change that added the writer: https://github.com/getsentry/snuba/pull/357

I don't think there was an intentional reason to make the `Reader` and `Writer` interfaces asymmetrical. (At least I don't recall making one intentionally.)

I think it would make sense longer-term for the `Writer` to act similarly to the `Reader` so that a single instance can be reused. I'm not totally sure about this, but that might need to happen as part of this change since this will create a new HTTP connection pool on each call to `get_writer`.

Reply:
I'm not really sure, but I suspect one reason for this is that the bulk_load script allows overriding the table name so it's not available on instantiation. This never happens with the reader.

Reply:
Yeah, we may want to consider reusing instances like the reader does. This PR doesn't currently change anything around this - we still have the same writer instances and connections established as before, it has just been rewired to go though the cluster and pick up the host and port values from there.